finished/src/main/java/com/example/android/wearable/composeforwearos/ReusableComponents.kt

```diff
@@ -225,13 +216,9 @@ fun ButtonExamplePreview() {
 
 // Text Preview
 @Preview(
```

Comment:
I'd suggest defining a WearPreview in this project and using that, rather than repeating. But can be a follow up.

Reply:
I considered that but decided to keep it simple for code lab. Definitely will need to use it in samples.